modules/apps/collaboration/message-boards/message-boards-web/src/main/resources/META-INF/resources/message_boards/edit_category.jsp

```diff
@@ -169,34 +169,34 @@ if (portletTitleBasedNavigation) {
 							<aui:input checked='<%= protocol.startsWith("imap") %>' label="imap" name="inProtocol" type="radio" value="imap" />
 						</aui:field-wrapper>
 
-						<aui:input label="server-name" name="inServerName" />
+						<aui:input disabled="true" label="server-name" name="inServerName" />
```

Comment:
We should see if we can leverage the backend to determine whether this field is disabled or not.  Inside the ```mailingList``` object, there is an ```active``` property which correlates to the active checkbox.  We should use that to determine whether this field is disabled or not.